src/DynamoPackagesUI/Utilities/CefCommands.cs

```diff
@@ -0,0 +1,68 @@
+﻿using CefSharp.Wpf;
+using Dynamo.DynamoPackagesUI.ViewModels;
+using Dynamo.Models;
+using Dynamo.PackageManager;
+using Dynamo.ViewModels;
+using Dynamo.Wpf.Interfaces;
+using DynamoPackagesUI.Utilities;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace Dynamo.DynamoPackagesUI.Utilities
+{
+    /// <summary>
+    /// Base class for CEF which assits in all Package Manager functionalities
+    /// </summary>
+    internal class CefCommands
+    {
+        internal DynamoPackagesUIClient Client { get; set; }
+
+        //internal readonly DynamoViewModel dynamoViewModel;
+        internal IBrandingResourceProvider ResourceProvider { get; set; }
```

Comment:
When you run it with Dynamo Studio or Dynamo Revit, this value is populated. If it's not there you can default it to `Dynamo`